README.md

```diff
@@ -54,7 +54,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - uses: dart-lang/setup-dart@v1.3
+      - uses: dart-lang/setup-dart@v1.4
```

Comment:
Maybe we can use `@v1`? /cc @mit-mit

Reply:
Yeah, that's what I would suggest users too.

That way they get the latest logic...